examples/performance-debugging/doc/results.dox

```diff
@@ -2,86 +2,90 @@
 This is the expected standard output:
 
 @code{.cpp}
-Solution, maximum first ten entries:
+
+Solution, first ten entries:
 [
-        0
-        0
-        0
-        0
-        0
-        0
-        0
-        0
-        0
-        0
+	0.252218
+	0.108645
+	0.0662811
+	0.0630433
+	0.0384088
+	0.0396536
+	0.0402648
+	0.0338935
+	0.0193098
+	0.0234653
 ];
 The performance and residual data can be found in log.txt
+
 @endcode
 
 Here is a sample output in the file log.txt:
 @code{.cpp}
-Generate time (ns): 3596
-Apply time (ns): 253224
+
+Generate time (ns): 30933
+Apply time (ns): 4367019
```

Comment:
the time is much slower than previous one. Could you rerun it when the system is idle? or original one is not on reference executor.

Reply:
This was done on my laptop with the reference executor, which is likely the reason why it is a bit slow.
Do you think the runtime matters here?